.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,61 +1,114 @@
+# Description
+<!--- Briefly describe the changes included in this pull request  --->
+
+### Closes #...  <!-- edit if this PR closes an Issue -->
+
+---
+## Test Results
+
+<!--- Please test the following in a built docker image.  --->
+
+
+### Validation Test
+
+#### BAM
+
+Case 1 - test type: <!-- e.g. fail on invalid, fail on empty, pass on valid, etc.  -->
+- input file(s):
+   ```
+   path/to/input
+   ```
+- command: 
+  ```
+  <command used>
+  ```
+- output: 
+  ```
+  path/to/output
```

Comment:
Since validation doesn't necessarily save to file, maybe we want something like `path/to/output OR output`?